cmd/demo/README.md

```diff
@@ -0,0 +1,62 @@
+## About
+
+This folder contains system test demo for the ws connector, which is going to receive exported data from a local testnet
+
+### How it works
+
+These scripts provide a demo to help you configure and test your ws connector/driver to receive data from a local
+testnet. The scripts found in this `demo` folder:
+
+- `local-testnet.sh` creates a local testnet with one shard(0) + metachain. Each shard has 2 nodes(1 validator + 1
```

Comment:
white space before `(` -> all over the file

Reply:
Right, added them